content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -1235,7 +1235,7 @@ contributors. Follow the instructions in the SIG-Windows
 * `kubectl port-forward` fails with "unable to do port forwarding: wincat not found"
 
   This was implemented in Kubernetes 1.15 by including wincat.exe in the
-  pause infrastructure container `mcr.microsoft.com/oss/kubernetes/pause:3.4.1`.
+  pause infrastructure container `k8s.gcr.io/pause:3.5`.
```

Comment:
This change is a little weird... I read this as saying the version of the pause image that _first_ had wincat.exe, so wouldn't have expected it to update with new releases. Perhaps it shouldn't be the full container repository name, but just mention the appropriate version? (Although `k8s.gcr.io/pause` has _always_ had Wincat, as long as it's supported Windows, i.e. 3.4, I assume the previous, distinct Windows pause image versions (1.x) are where that was first introduced.)

Reply:
wincat was introduced in `mcr.microsoft.com/k8s/core/pause:1.2.0` ([reference](https://github.com/kubernetes/website/pull/15345/files)), but that line's been updated several times since, so I guess "Be sure to use these versions or newer ones." hasn't been a strict requirement for a while, and can probably be removed.

Perhaps it should not bother naming the container image version which is adequately described elsewhere, and just say
> This was implemented in Kubernetes 1.15 by including wincat.exe in the pause infrastructure container.

since the following paragraph explains what to do if you're _not_ using a build of the k8s-specified pause image.

Reply:
Yea, i'll make these updates.
IIRC this did not specify a specific version of the pause image and then there was feedback saying we should include a version of the pause image for clarity. It has been so long now that all of the pause images folks can find should have wincat.exe (which hasn't changed) so not it makes sense to just drop the specific reference.

Reply:
I tried to re-word this. Let me know if it is more clear (and thanks for the feedback!)

Reply:
Yeah, looks good and much clearer than before you started. Thank you.